jenkins/deploy/Jenkinsfile

```diff
@@ -8,15 +8,19 @@ pipeline {
   }
 
   parameters {
-    choice(name: 'RELEASE_TYPE', choices: ['M0', 'M1', 'M2', 'M3', 'RC1', 'RC2', 'GA'], description:
+    choice(name: 'DEPLOYMENT_TYPE', choices: ['nightly' /*first is default*/, 'M0', 'M1', 'M2', 'M3', 'RC1', 'RC2', 'GA'], description:
       '''
         Kind of milestone release to build.
       ''')
   }
 
+  triggers { // https://jenkins.io/doc/book/pipeline/syntax/#triggers
+    cron('50 21 * * *') // nightly at 21:50
+  }
+
```

Comment:
I tested this in my local Jenkins instance and it worked as desired. The job run automatically at the specified time with `DEPLOYMENT_TYPE=nightly`, and if one trigger an execution manually one has to choose one of the available parameter values.